### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by richsalz**
Not planning to backport (doc-nits is only in master) unless there's pressure.

### CODE REVIEW

File: CONTRIBUTING [hunk 1]
```diff
@@ -1,26 +1,27 @@
-HOW TO CONTRIBUTE PATCHES TO OpenSSL
+HOW TO CONTRIBUTE TO PATCHES OpenSSL
 ------------------------------------
 
 (Please visit https://www.openssl.org/community/getting-started.html for
 other ideas about how to contribute.)
 
-Development is coordinated on the openssl-dev mailing list (see the
+Development is coordinated on the openssl-project mailing list (see the
```
Comment (levitte) [inline CONTRIBUTING]: No.  openssl-project is not intended to be a new openssl-dev.  We should say that we're coordinating development on github.

File: CONTRIBUTING [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
 above link or https://mta.openssl.org for information on subscribing).
 If you are unsure as to whether a feature will be useful for the general
-OpenSSL community you might want to discuss it on the openssl-dev mailing
+OpenSSL community you might want to discuss it on the openssl-project mailing
```
Comment (levitte) [inline CONTRIBUTING]: No.  You raise an issue on github.

File: CONTRIBUTING [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
 list first.  Someone may be already working on the same thing or there
 may be a good reason as to why that feature isn't implemented.
 
 To submit a patch, make a pull request on GitHub.  If you think the patch
-could use feedback from the community, please start a thread on openssl-dev
+could use feedback from the community, please start a thread on openssl-project
```
Comment (levitte) [inline CONTRIBUTING]: No, just submit a PR on github, discussion will happen there.

**Comment by levitte** [comment]
I would also very much like to see whatever changes we end up with backported to 1.1.0, as much as is appropriate (yes, I know, doc-nits is master only, but the rest is the same for all branches)

**Comment by t-j-h** [comment]
As I noted in the other PR - we need to be clear to point what used to go to openssl-dev off to github as per OMC decisions. openssl-project is not a replacement for openssl-dev - it has an entirely different purpose.

And the elimination of RT references should be backported for active release (1.0.2, 1.1.0) ...

File: CONTRIBUTING [hunk 4]
```diff
@@ -1,26 +1,26 @@
-HOW TO CONTRIBUTE PATCHES TO OpenSSL
+HOW TO CONTRIBUTE TO PATCHES OpenSSL
```
Comment (mspncp) [inline CONTRIBUTING]: Did you change this line intentionally? This does not sound like correct english to me.

File: CONTRIBUTING [hunk 5]
```diff
@@ -1,26 +1,26 @@
...
 
-Having addressed the following items before the PR will help make the
-acceptance and review process faster:
+To submit a patch, make a pull request on GitHub.  If you are thinking of
+making a large contribution, open an issue for it before starting work, to
+get comments from the community.  Someone may be already working on the same
+thing or there may be reasons why that feature isn't implemented.
 
-    1. Anything other than trivial contributions will require a contributor
+To make it easier to review and accept our pull request, please follow these
```
Comment (mspncp) [inline CONTRIBUTING]: 'our pull request' -> 'your pull request'

File: CONTRIBUTING [hunk 6]
```diff
@@ -1,26 +1,26 @@
...
+guidelines:
+
+    1. Anything other than a trivial contribution requires a contributor
     licensing agreement, giving us permission to use your code. See
```
Comment (mspncp) [inline CONTRIBUTING]: It's better to use the formulation of [cla.html](https://www.openssl.org/policies/cla.html), which calls it a "Contributor License Agreement (CLA)".

File: CONTRIBUTING (hunk 4 above)
Comment (mspncp) [reply CONTRIBUTING]: Maybe you intended to write "HOW TO CONTRIBUTE TO OpenSSL"?

File: CONTRIBUTING [hunk 7]
```diff
@@ -34,21 +34,22 @@ acceptance and review process faster:
...
 
     5.  When at all possible, patches should include tests. These can
     either be added to an existing test, or completely new.  Please see
     test/README for information on the test framework.
 
     6.  New features or changed functionality must include
-    documentation. Please look at the "pod" files in doc/man[1357]
-    for examples of our style.
+    documentation. Please look at the "pod" files in doc/man[1357] for
+    examples of our style. Do "make doc-nits" to make sure that your
```
Comment (mspncp) [inline CONTRIBUTING]: Maybe replace "Do" -> "Run" ?

File: CONTRIBUTING [hunk 8]
```diff
@@ -1,26 +1,26 @@
...
+thing or there may be reasons why that feature isn't implemented.
 
-    1. Anything other than trivial contributions will require a contributor
-    licensing agreement, giving us permission to use your code. See
-    https://www.openssl.org/policies/cla.html for details.
+To make it easier to review and accept your pull request, please follow these
+guidelines:
+
+    1. Anything other than a trivial contribution requires a Contributor
+    Licensing Agreement (CLA), giving us permission to use your code. See
```
Comment (mspncp) [inline CONTRIBUTING]: Nit: the official wording is „License“, not „Licensing“, see the cla pdf.

**Comment by richsalz** [comment]
I'll wait until @t-j-h has a chance to look before pushing.  I will cherry-pick, by just copying the file over to each release (and deleting the doc-nits sentence for the non-master branches).

File: CONTRIBUTING (hunk 8 above)
Comment (richsalz) [reply CONTRIBUTING]: fixed.

**Comment by richsalz** [comment]
Thanks for your feedback.  You don't have to abstain :)  Tim had made excellent points, too, and I want to make sure this addresses them before I merge.

File: CONTRIBUTING [hunk 9]
```diff
@@ -1,26 +1,26 @@
...
-If you are unsure as to whether a feature will be useful for the general
-OpenSSL community you might want to discuss it on the openssl-dev mailing
-list first.  Someone may be already working on the same thing or there
-may be a good reason as to why that feature isn't implemented.
+Development is done on GitHub, https://github.com/openssl/openssl.
 
-To submit a patch, make a pull request on GitHub.  If you think the patch
-could use feedback from the community, please start a thread on openssl-dev
-to discuss it.
+To request new features or report bugs, please open an issue.
```
Comment (FdaSilvaYY) [inline CONTRIBUTING]: ... an issue on GitHub.

File: CONTRIBUTING [hunk 10]
```diff
@@ -1,26 +1,26 @@
...
 
-Having addressed the following items before the PR will help make the
-acceptance and review process faster:
+To submit a patch, make a pull request on GitHub.  If you are thinking of
```
Comment (FdaSilvaYY) [inline CONTRIBUTING]: "To submit a patch, please open a pull request on GitHub..."

Comment (FdaSilvaYY) [reply CONTRIBUTING]: There is some steps before opening a PR that can be mentionned, like cloning the OpenSSL repository, create our work branch, push the change onto it, and them create a PR with this edited branch.

File: CONTRIBUTING [hunk 11]
```diff
@@ -34,21 +34,22 @@ acceptance and review process faster:
...
     4.  Patches should follow our coding style (see
-    https://www.openssl.org/policies/codingstyle.html) and compile without
-    warnings. Where gcc or clang is available you should use the
+    https://www.openssl.org/policies/codingstyle.html) and compile
+    without warnings. Where gcc or clang is available you should use the
     --strict-warnings Configure option.  OpenSSL compiles on many varied
-    platforms: try to ensure you only use portable features.
-    Clean builds via Travis and AppVeyor are expected, and done whenever
-    a PR is created or updated.
+    platforms: try to ensure you only use portable features.  Clean builds
```
Comment (FdaSilvaYY) [inline CONTRIBUTING]: "Clean builds..."  deserve a paragraph of its own, with some word on creating/opening a personal Travis/AppVeyor account ( very quick stepo using own GitHub login account) , sot anyone can push and test there changes without opening a PR.

**Comment by mspncp** [comment]
@FdaSilvaYY, you propose a lot of good ideas here, but some of them go beyond the scope of a simple CONTRIBUTING text file. For example, it can't be purpose of it to provide a complete GitHub tutorial. There are are a lot of good tutorials out there, not only [on GitHub](https://guides.github.com).

I agree that it would be a good thing to have a place were OpenSSL specific GitHub matters like the basic workflow and the build serves are explained. But a plain text format is not the ideal medium for doing that. The Wiki would be a more suitable location.

An even better and more visible solution (than a link to the wiki) is the possibility to create a welcome page for the project. I happened to see a nice one just recently on the [google/tink](https://github.com/google/tink) project page. If you [scroll down a little](https://github.com/google/tink#tink), you see it. The source file, [README.md](https://github.com/google/tink/blob/master/README.md) is part of the repository and written in [github flavoured markdown](https://raw.githubusercontent.com/google/tink/master/README.md). From the welcome page, one could link into further useful documentation related to OpenSSL on GitHub, which could be located in docs/github as markdown files. There you would have a lot of possibilities to follow your ideas...

File: CONTRIBUTING (hunk 9 above)
Comment (richsalz) [reply CONTRIBUTING]: ok

File: CONTRIBUTING (hunk 10 above)
Comment (richsalz) [reply CONTRIBUTING]: I took the wording change, but this is not the place for a GitHub tutorial, so not going to do that.  Having that kind of doc, probably on our website, makes sense.

File: CONTRIBUTING (hunk 11 above)
Comment (richsalz) [reply CONTRIBUTING]: Again, I think this should be a separate doc on the website.

**Comment by richsalz** [comment]
I know @t-j-h is busy this week with RSA, so I'll merge in a couple of days anyway.

**Comment by richsalz** [comment]
Merged to all three branches (sans the doc-nits sentence for non-master); thanks everyone for the feedback.

**Pull request closed without merging**